packages/design-system/src/components/tooltip.tsx

```diff
@@ -13,7 +13,7 @@ import { Box } from "./box";
 import { Text } from "./text";
 import type { CSS } from "../stitches.config";
 import { theme } from "../stitches.config";
-import { disableCanvasPointerEvents } from "../utilities";
+import { composeEventHandlers } from "@radix-ui/primitive";
```

Comment:
We already have such utility here
![image](https://github.com/user-attachments/assets/2cc36c31-f2dc-48d3-8f32-b52345ed61f0)